app/config/parameters.yml.dist

```diff
@@ -14,7 +16,5 @@ parameters:
     mailer_user:       ~
     mailer_password:   ~
 
-    locale:            en
-
     # A secret key that's used to generate certain security-related tokens
     secret:            ThisTokenIsNotSoSecretChangeIt
```

Comment:
Thanks Tobion - I see the comment here: https://github.com/symfony/symfony/issues/14026#issuecomment-85030221

I don't think this should hold up this PR, though - I think moving the secret somewhere else should be a different issue.